### DISCUSSION TITLE
Will platform XAML be support after WinUI 3

### DISCUSSION THREAD

**Discussion by jaigak** [microsoft/WindowsAppSDK#220] [General] [answered]
Will platform XAML (Windows.UI.Xaml) still be supported after WinUI 3 and will Microsoft update the styles to match WinUI because today the platform XAML apps don't have things like rounded corners. Also, will Visual Studio in the future provide templates that use platform XAML instead of WinUI 3?

**Comment by maxkatz6** [comment]
> XAML apps don't have things like rounded corners

Today developers can use WinUI 2 for this. And as I know, it will be supported and updated for a while after WinUI 3 release. Although I would like to know official information about it.

**Comment by mdtauk** [comment]
I believe the current Windows 10 platform XAML is in maintenance mode, and WinUI 2.X will get the new controls and styling.

Once WinUI 3 is out, the intention is for people to move to it, and I think there is tooling being planned to handle most of the conversion from WinRT UWP to WinUI 3 UWP

**Comment by Felix-Dev** [comment, marked as answer]
"The existing UWP XAML APIs that ship as part of the OS will no longer receive new feature updates. They will still receive security updates and critical fixes according to the Windows 10 support lifecycle."

https://github.com/microsoft/microsoft-ui-xaml/blob/master/docs/roadmap.md#winui-3
